package.json

```diff
@@ -950,25 +950,68 @@
+                                                        "enum": [
+                                                            "linux",
+                                                            "darwin",
+                                                            "windows"
+                                                        ]
+                                                    },
+                                                    {
+                                                        "type": "string"
+                                                    }
+                                                ],
+                                                "description": "%vscode-docker.tasks.docker-build.dockerBuild.platform.os%"
+                                            },
+                                            "architecture": {
+                                                "anyOf": [
+                                                    {
+                                                        "type": "string",
+                                                        "enum": [
+                                                            "amd64",
+                                                            "386",
+                                                            "arm64",
+                                                            "arm"
+                                                        ]
+                                                    },
+                                                    {
+                                                        "type": "string"
+                                                    }
+                                                ],
+                                                "description": "%vscode-docker.tasks.docker-build.dockerBuild.platform.architecture%"
+                                            }
+                                        }
                                     },
                                     {
                                         "type": "string",
-                                        "enum": [
-                                            "linux/amd64",
-                                            "linux/386",
-                                            "linux/arm64",
-                                            "linux/arm",
-                                            "darwin/amd64",
-                                            "darwin/386",
-                                            "darwin/arm64",
-                                            "darwin/arm",
-                                            "windows/amd64",
-                                            "windows/386",
-                                            "windows/arm64",
-                                            "windows/arm"
+                                        "anyOf": [
```

Comment:
This "anyOf" isn't necessary if you specify an "anyOf" condition for "platform". It should be "anyOf" the object, generic string type, or specific enum type.